chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/surveillance/SurveillanceValidator.java

```diff
@@ -349,4 +355,26 @@ public void validateSurveillanceNonconformities(Surveillance surv) {
 			}
 		}
 	}
+	
+	public void validateSurveillanceAuthority(Surveillance surv) {
```

Comment:
I'm envisioning the Controller works pretty much as you have it and either the Authority is put on the Surveillance object by the UI or you fill it in based on the user's role. Then validate() is called from the Controller which would check that the Authority is indeed filled in (you're right - it should also check that the authority value is valid based on the roles the current user has and the possible choices but it can still be simplified from what is there). The Manager would be where roles are checked... so in the survManager.create() method, as long as the validator passed in the controller you'd know the surveillance has an allowed authority and can be created. The Manager update method is slightly more complicated only in that you need to check that the user performing the update isn't ACB_ADMIN if the Authority in the existing surveillance db row is ONC_ADMIN (if ONC created the surveillance then ACB can't be the one to update it) - basically just check that the user performing the update is allowed to.